source/code/source/playfab/PlayFabIXHR2HttpRequest.cpp

```diff
@@ -11,6 +11,8 @@
 
 #include <stdafx.h>
```

Comment:
Same here, you may move it under

Reply:
Repeat, I disagree, and it's likely a compile error.